tracekit.js

```diff
@@ -233,11 +233,10 @@ TraceKit.report = (function reportModuleWrapper() {
      */
     function processLastException() {
         var _lastExceptionStack = lastExceptionStack,
-            _lastArgs = lastArgs;
-        lastArgs = null;
+            _lastException = lastException;
```

Comment:
I think our goal here is to preserve compatibility (as per [this comment](https://github.com/csnover/TraceKit/issues/66#issuecomment-269486631)), so I chose to leave the second argument (`isWindowError`) intact.